src/webui/webui.c

```diff
@@ -410,11 +428,17 @@ http_stream_run(http_connection_t *hc, profile_chain_t *prch,
       break;
 
     case SMT_GRACE:
-      grace = sm->sm_code < 5 ? 5 : grace;
+      if (sm->sm_code > 5) {
```

Comment:
I'm not quite sure what is the intent of the old logic, so I have preserved it here.

In my testing, playing every stream results in a `SMT_GRACE` with a code of 3 on startup, which ends up being ignored.